### ISSUE TITLE
When I press spacebar to pause/play, it overlays several duplicate tracks of the song

### ISSUE THREAD

**Issue by jimhavrilla** [radiant-player/radiant-player-mac#544]
Basically I disabled the iTunes play/pause launch ability with a .dmg patch so I can use the play/pause button and have it just work for Radiant Player.  The issue is if I try to stop Radiant Player using spacebar even by mistake, pure instinct this aforementioned error occurs where the song plays over and over offset by the number of seconds paused basically.

**Comment by jacobwgillespie** [comment]
Can you reproduce this in Safari?  (like, open GPM in Safari and try to replicate)

**Comment by jimhavrilla** [comment]
No, it just happens in Radiant Player.  I use Chrome and it doesn't happen there.  Just tried Safari too and not there either.
